plugins/browser-plugin-ga-cookies/src/types.ts

```diff
@@ -0,0 +1,14 @@
+export interface UACookies {
+  __utma?: string;
+  __utmb?: string;
+  __utmc?: string;
+  __utmv?: string;
+  __utmz?: string;
+  _ga?: string;
+  [key: string]: string | undefined;
```

Comment:
Any idea why this `key` thing is defined?

Reply:
I think this was about making typescript happy in the loop of the index file _(not in the most TypeScripty way)_.